docs: add Artiel release (#21)
skyra-project/blog.skyra.pw: content/6.artiel-release.md

---
title: Artiel's release
description: Skyra Project's Fun bot just released!
date: 2023-06-06
tags: [first-release]
author: Aura
image: artiel
---

**Artiel is Skyra Project's Fun bot**, he comes with a growing database of meme templates powered by our custom-made
[Meme Template Generator](https://memes.skyra.pw).

::card{type="info"}

> Right now, the meme template database will start small, but will soon be filled with lots of templates. We tried to use
> an external database, but not only they are locked down and require payment for accessing the list, but they also come
> with a lot of duplicates, something we aim to fix with our tool.

Reviewer: ```suggestion
Right now, the meme template database will start small, but we aim to soon provide lots of templates.
```

The whole second sentence is unnecessary. No one will care about such fine details.

  with the ability to wrap text within a box rather than within a horizontal width boundary, this allows Artiel to
  generate memes with virtually any template in existence while also supporting placing avatars on the template. Because
  of this, and other limitations, we had to build an [entire website](https://memes.skyra.pw) open to the community to
  aid with the creation of said templates.
- `/ship`: In Skyra, the theme used for the command is defined by whether or not the dark theme was enabled, supporting
  only two themes: light and dark. Artiel by comparison, expands this by adding more themes and having a higher image
  resolution.
- `/xkcd`: The search of the comic by name has been broken for a long time, this is due to the third-party API we were
  using being shut down by its developers. Artiel restores this feature by storing a local database of all the comics on
  xkcd, which is indexed at 12:00 UTC on weekdays (Randall Munroe, the author, doesn't post on weekends). On top of it,
  Artiel's search includes descriptions, which makes results even easier to find.

### Wait, but didn't you tell us the bot would have games and such?

> Yes, we did, and Artiel will eventually have games, we wanted a first version out so the community can have better memes
> than the ones available in Skyra. Worry not, Artiel will feature a better component AI to play against and a better
> command design so it's not tedious to play on.

Reviewer: ```suggestion
Yes, we did, and Artiel will eventually have games. We wanted a first version out so the community can have better memes
than the ones available in Skyra. Worry not, Artiel will feature a better AI to play against and a better
command design so it's not tedious to play the games.
```

> We didn't want to rush features, so instead, we'll ship them once it's properly done and polished, the game commands
> will continue in Skyra in the meantime.

Reviewer: ```suggestion
We didn't want to rush features, so instead, we'll ship them once they're properly done and polished. The game commands will remain a part of Skyra in the meantime.
```

### When can we expect Artiel to have games?

> We cannot say for sure, we will probably release as they are done and tested. At the earliest, the first game may be
> shipped in at least a month.

Reviewer: ```suggestion
We cannot say for sure, we will probably release as they are done and tested.
```

Best to just not make any promises tbh
